### PR TITLE
More HTTP/2 queries

### PR DESCRIPTION
**Pull request by gregorywolf**
Makes more progress on #921

### CODE REVIEW

File: sql/2020/22_HTTP_2/cdn_summary.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  percentile,
+  client,
+  firstHTML,
+  CDN
+ORDER BY
+  percentile,
+  client,
+  firstHTML,
+  CDN
+
```
Comment (bazzadp) [inline sql/2020/22_HTTP_2/cdn_summary.sql:44]: Nit - remove the spacing to finsih on a blank line.

```suggestion

```

I dunno why that red mark bugs me so, in GitHub but it does!

No new line at end marker

File: sql/2020/22_HTTP_2/http2_%_1st_party_vs_3rd_party.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+UNNEST([5, 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, 100]) AS percentile
+GROUP BY
+  percentile,
+  client,
+  is_third_party
+ORDER BY
+  percentile,
+  client,
+  is_third_party
+
```
Comment (bazzadp) [inline sql/2020/22_HTTP_2/http2_%_1st_party_vs_3rd_party.sql:42]:
```suggestion

```

**Review by bazzadp — APPROVED** [review]
Approved. @rviscomi are you OK with non-standard percentiles [given my rational here](https://github.com/HTTPArchive/almanac.httparchive.org/issues/921#issuecomment-722025909)?

**Comment by rviscomi** [comment]
LGTM. I'm not 100% on board with it but don't want to hold you up.

**Review by rviscomi — APPROVED** [review]

**Pull request merged**